services/src/main/java/com/linkedin/kmf/services/configs/ConsumeServiceConfig.java

```diff
@@ -42,6 +42,10 @@
   public static final String CONSUMER_PROPS_CONFIG = "consume.consumer.props";
   public static final String CONSUMER_PROPS_DOC = "The properties used to config consumer in consume service.";
 
+  public static final String LATENCY_SLA_MS_CONFIG = "consume.latency.sla.ms";
```

Comment:
There is reason for have the prefix `consume` in the config. For example, we have these two configs in `ConsumeServiceConfig` - `consume.latency.percentile.max.ms` and `consume.consumer.props`. And we have these two configs in `ProduceServiceConfig` -  `produce.thread.num` and `produce.thread.num`. The prefix `consume.*` (or `produce.*`) is used to indicate which service this config belongs to. `consume.latency.percentile.max.ms` is directly used for that ConsumeService, while `consume.consumer.props` is used for the consumer in the ConsumeService. It is because of this kind of structure that we have named `consume.consumer.* ` instead of simply `consumer.*`.